src/backend/joanie/core/api/admin.py

```diff
@@ -111,6 +114,19 @@ def get_queryset(self):
 
         return super().get_queryset()
 
+    @action(
+        detail=False,
+        methods=["get"],
+        permission_classes=[permissions.IsAuthenticated],
+    )
+    # pylint: disable=invalid-name
+    def me(self, request):
+        """
+        Return information on currently logged user
+        """
+        user = self.queryset.get(username=request.user)
+        return Response(self.me_serializer_class(user).data)
```

Comment:
I don't understand how this get works :thinking: 
Shouldn't it be something like:
```suggestion
        if request.user.is_authenticated:
            return Response(self.me_serializer_class(request.user).data)
        raise AuthenticationError (or something like this to raise a 401?)
```

Reply:
In the decorator `@action` there is `permission_classes=[permissions.IsAuthenticated]` which checks that the user is authenticated or returns 403

Reply:
Ok! Thanks. Can you add a test for anonymous users.

The other thing I didn't understand is how "username=request.user" works to filter the user. Shouldn't it be "user=request.user"?